botorch/optim/initializers.py

```diff
@@ -367,7 +368,7 @@ def gen_batch_initial_conditions(
                     q=q,
                     bounds=bounds,
                     n_burnin=options.get("n_burnin", 10000),
-                    thinning=options.get("thinning", 32),
+                    n_thinning=options.get("n_thinning", 32),
```

Comment:
If someone passes in `thinning`, I am guessing it'll get silently ignored. Would something like this offer better backwards compatibility? Ideally we'd just error out if we got some unsupported option cc @esantorella 
```suggestion
                    n_thinning=options.get("n_thinning", options.get("thinning", 32)),
```